spaceship/lib/spaceship/tunes/member.rb

```diff
@@ -24,6 +24,14 @@ def roles
         return parsed_roles
       end
 
+      def admin?
+        roles.include?("admin")
+      end
+
+      def app_manager?
+        roles.include?("appmanager")
+      end
+
```

Comment:
`roles` looks like they could be an enumeration. Let's have them extracted like this: https://github.com/fastlane/fastlane/blob/master/spaceship/lib/spaceship/test_flight/build.rb#L67-L73